src/core/text/Text.js

```diff
@@ -130,48 +131,11 @@ export default class Text extends Sprite
 
         this._font = Text.getFontStyle(style);
 
-        this.context.font = this._font;
-
-        // word wrap
-        // preserve original text
-        const outputText = style.wordWrap ? this.wordWrap(this._text) : this._text;
-
-        // split text into lines
-        const lines = outputText.split(/(?:\r\n|\r|\n)/);
-
-        // calculate text width
-        const lineWidths = new Array(lines.length);
-        let maxLineWidth = 0;
-        const fontProperties = Text.calculateFontProperties(this._font);
-
-        for (let i = 0; i < lines.length; i++)
-        {
-            const lineWidth = this.context.measureText(lines[i]).width + ((lines[i].length - 1) * style.letterSpacing);
-
-            lineWidths[i] = lineWidth;
-            maxLineWidth = Math.max(maxLineWidth, lineWidth);
-        }
-
-        let width = maxLineWidth + style.strokeThickness;
-
-        if (style.dropShadow)
-        {
-            width += style.dropShadowDistance;
-        }
+        const measured = TextMetrics.measure(this._text, this._style, this._style.wordWrap, this.canvas);
```

Comment:
The advantage is in the minified file size and performance. If you have multiple instances, it's a little better to cache the variable. here's an example:

```js
const measure = TextMetrics.measure();
const width = measure.width;
const height = measure.height;
```
Would minify to:
```js
const a = TextMetrics.measure();
const b = a.width;
const c = a.height;
````

The more you reference `measure.width`, etc the less compressed the source becomes after minification and the more times it needs to use the dot-accessor.